src/components/calcite-action/calcite-action.scss

```diff
@@ -140,9 +120,7 @@
   background-color: var(--calcite-app-background-active);
 }
 
-:host([indicator]) .button--text-visible-visible,
-:host([indicator]) .button--text-interactive:focus,
-:host([indicator]) .button--text-interactive:hover {
+:host([indicator]) .button--text-visible-visible {
```

Comment:
Nitpick: unrelated to this PR, but is there another name we could use for this modifier to avoid the duplicate `visible`?

Reply:
I think it should actually be `button--text-visible` so likely a bug? @asangma

Reply:
Updated